ml-agents-envs/mlagents_envs/tests/test_gym_wrapper.py

```diff
@@ -0,0 +1,44 @@
+from mlagents_envs.gym_wrapper import GymToUnityWrapper
+from mlagents_envs.base_env import ActionType
+
+try:
+    import gym
+except ImportError:
+    raise ImportError(
+        "gym is not installed, please call `pip install gym` before running this test"
+    )
+import pytest
+
+GYM_ENVS = ["CartPole-v1", "MountainCar-v0"]
+
+
+@pytest.mark.parametrize("name", GYM_ENVS, ids=GYM_ENVS)
+def test_creation(name):
+    env = GymToUnityWrapper(gym.make(name), name)
+    env.close()
+
+
+@pytest.mark.parametrize("name", GYM_ENVS, ids=GYM_ENVS)
+def test_specs(name):
+    gym_env = gym.make(name)
+    env = GymToUnityWrapper(gym_env, name)
+    assert env.get_behavior_names()[0] == name
+    if isinstance(gym_env.action_space, gym.spaces.Box):
```

Comment:
I don't think this is a meaningful test, since you're just duplicating the logic from the code.

A better approach would be having an expected ActionType for each gym env and making sure that's set correctly.